nrf54h20: Fix EXMIF build failures

krish2718 opened the pull request:
Build failure started with this https://github.com/zephyrproject-rtos/zephyr/commit/fa86c518f6704c9d782cf2bbce4808cc6ee01a77

krish2718 committed:
Revert "samples: drivers: spi_flash: Add nRF54H20DK support"

This reverts commit 064d8d8a1a580e366dbd322fc4702f0a3910e6e5.

This causes build failure, EXMIF support for nRF54H20DK needs mor
 work.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

krish2718 committed:
Revert "samples: drivers: jesd216: add nRF54H20 overlay"

This reverts commit b1de9a6c46217b4edc7befb928a9cb2e6d010ca3.

This causes build failure, EXMIF support for nRF54H20DK needs more
work.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

JordanYates commented:
Pulled both of these revert commits into #73302, which fixes both issues with the sample in one PR, letting CI pass.

krish2718 committed:
boards: st: Fix SPI driver sample

Add the necessary alias for SPI flash sample to fix the build error. The
sample auto-chooses the device based on DT compatible property, so,
having the alias mandatory.

Fixes commit c8faa1e3809("boards: st: Add support for
STM32H745I-Disco").

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

FRASTM commented:
Seems taht stm32 label added by the  [zephyrbot] is not relevant, is it

FRASTM commented on boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m7.dts:
```diff
@@ -46,6 +46,7 @@
 		led0 = &green_led;
 		pwm-led0 = &green_pwm_led;
 		sw0 = &user_button;
+		spi-flash0 = &mt25ql512ab1;
```
this part of the PR # https://github.com/zephyrproject-rtos/zephyr/pull/73027/

krish2718 commented:
Closing PR as all fixes are part of other PRs linked above.
